src/commands/Moderation/addRole.ts

```diff
@@ -0,0 +1,29 @@
+import { checkModeratable, getTargetDM } from '@lib/structures/ModerationCommand';
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { PermissionLevels } from '@lib/types/Enums';
+import { ApplyOptions } from '@skyra/decorators';
+import { Role, User } from 'discord.js';
+import { KlasaMessage } from 'klasa';
+
+@ApplyOptions<SkyraCommandOptions>({
+	cooldown: 10,
+	description: language => language.tget('COMMAND_ADDROLE_DESCRIPTION'),
+	extendedHelp: language => language.tget('COMMAND_ADDROLE_EXTENDED'),
+	requiredGuildPermissions: ['MANAGE_ROLES'],
+	permissionLevel: PermissionLevels.Administrator,
+	usage: '<user:username> <role:rolename> [reason:...string]',
```

Comment:
Any reason to not support duration? AddRoles can be reversed (removing the role)